server/api/models/company.js

```diff
@@ -0,0 +1,52 @@
+var mongoose = require('mongoose');
+var db = require('./db');
+
+var CompanySchema = new db.Schema({
+  companyName: {
+    type: String,
+    required: true
+  },
+  address: {
+    address1: {
+      type: String,
+      required: true
+    },
+    address2: {
+      type: String,
+      required: true
+    },
+    city: {
+      type: String,
+      required: true
+    },
+    state: {
+      type: String,
+      required: true
+    },
+    country: String,
+    zip: {
+      type: String,
+      required: true
+    }
+  },
+  clients: [{
```

Comment:
The client is basically another company under the main company. I think every client will also need an address and possibly a department, po number, and possibly other info.